tests/masonry/builders/fixtures/licenses_and_copying/pyproject.toml

```diff
@@ -43,7 +43,6 @@ time = ["pendulum"]
 [tool.poetry.scripts]
 my-script = "my_package:main"
 my-2nd-script = "my_package:main2"
-extra-script = {reference = "my_package.extra:main", extras = ["time"], type = "console"}
```

Comment:
How about we deprecate the dict/table syntax all together? With extras deprecation, it makes no sense to keep it around, since we don't support any other type than `console`.

Reply:
Sounds good to me.